documentation/gcviewer.asciidoc

```diff
@@ -0,0 +1,26 @@
+:toc:
+toc::[]
+
+= GCViewer
+
+The `gcviewer` commandlet is used to install the GCViewer App if not already installed or launch the App if already installed.
+GCViewer is a little tool that visualizes verbose GC (Garbage Collection) output generated by Java Virtual Machines.
+To learn more about GC Loggings, please refer to https://sematext.com/blog/java-garbage-collection-logs/ or https://www.baeldung.com/java-gc-logging-to-file.
+
+*ATTENTION:*
+Currently this feature is new and therefore experimental.
+It may change in incompatible ways in the next releases until we reach a stable state.
+We hope that all is working fine for you.
+However, do not expect everything to work out of the box.
+In case you are facing issues (e.g. network problems with Cisco AnyConnect, etc.) please give us feedback so we can improve.
+
+
+The arguments (`devon oc «args»`) are explained in the following table:
```

Comment:
should be `gcviewer` instead of `oc`. Seems you copy & pasted this from openshift CLI :)